addons/web/static/tests/views/graph/graph_view.test.js

```diff
@@ -39,12 +39,11 @@ import {
     getGraphRenderer,
     getModeButton,
     getScaleY,
-    getXAxisLabel,
     getYAxisLabel,
```

Comment:
the test with getYAxisLabel are successful but the title is not visible in the charts.
Beside that, I am not sure it is good to remove the x label. The graph groupBy is not always visible in the search bar (if it comes from arch).

Reply:
If you nee to remove one or both of the helpers getXAxisLabel/getYAxisLabel, you should remove them from
the file graph_test_helpers.js